src/etc/inc/system.inc

```diff
@@ -616,25 +616,26 @@ function system_dhcpleases_configure() {
 			$unbound_conf = "";
 		}
 
-		if (isvalidpid($pidfile)) {
-			/* Make sure dhcpleases is using correct unbound or dnsmasq */
-			$_gb = exec("/bin/pgrep -F {$pidfile} -f {$dns_pid}", $output, $retval);
-			if (intval($retval) == 0) {
-				sigkillbypid($pidfile, "HUP");
-				return;
-			} else {
-				sigkillbypid($pidfile, "TERM");
-			}
-		}
+		/* best to never send a HUP to dhcpleases, it is bugged */
+		// if (isvalidpid($pidfile)) {
+		// /* Make sure dhcpleases is using correct unbound or dnsmasq */
+		//	$_gb = exec("/bin/pgrep -F {$pidfile} -f {$dns_pid}", $output, $retval);
+		//	if (intval($retval) == 0) {
+		//		sigkillbypid($pidfile, "HUP");
+		//		return;
+		//	} else {
+		//		sigkillbypid($pidfile, "TERM");
+		//	}
+		// }
```

Comment:
If the code is useless, just remove it instead of commenting it out

Reply:
Yes, true. I will submit an update to remove this.

Reply:
The `pkill -x` was to ensure that all instances were killed to avoid multiple instances (which is something I observed happening occasionally while I was debugging). But, if you prefer to use existing functions then I suppose `sigkillbyname('dhcpleases', "TERM")` does the same thing.

My thinking was that just killing it was a little faster than first calling `is_process_running()` and then `sigkillbyname()` since in the end we are just trying to kill all instances, regardless of whether 0, 1 or 10 of these procs exist.